lib/daru/io/importers/linkages/plaintext.rb

```diff
@@ -0,0 +1,23 @@
+require 'daru'
+
+module Daru
+  class DataFrame
+    class << self
+      # Read the database from a plaintext file. For this method to work,
+      # the data should be present in a plain text file in columns. See
+      # spec/fixtures/bank2.dat for an example.
+      #
+      # == Arguments
+      #
```

Comment:
Thanks! I've now added `yard` as a development dependency. Once the porting is done as-is with better spec styling, I'll rewrite with better YARD docs as per the guide, by tomorrow. 😄